src/settings.py

```diff
@@ -13,7 +13,9 @@
     cast=lambda filename: tomllib.loads(Path(filename).read_text()),
     default={},
 )
-LLM_CONFIG = PROJECT_CONFIG.get("llm") or {"temperature": 0.2}
+
+DEFAULT_LLM_CONFIG = {"temperature": 0.2, "model_name": "gpt-3.5-turbo"}
+LLM_CONFIG = PROJECT_CONFIG.get("llm") or DEFAULT_LLM_CONFIG
```

Comment:
Making this way, we force the user to provide all LLM configs. I think that we can use `DEFAULT_LLM_CONFIG` as a base and override it with user provided config.

We can achieve this merging the two dicts
```python
DEFAULT_LLM_CONFIG = {"temperature": 0.2, "model_name": "gpt-3.5-turbo"}
LLM_CONFIG = {**DEFAULT_LLM_CONFIG, **PROJECT_CONFIG.get("llm", {})}
```

Reply:
hmm I did not anticipate that, tnks. I'll change here.

Reply:
there is a drawback for the future, however: this will only work as long as we keep openai's gpt as the llm model. If, in the future, we plan to allow other llms, maybe we'll need a different logic (e.g. adding defaults for each llm allowed and some logic to first find which llm to use). 

For now I think we don't need to add such complexity, so your idea is good to avoid extra work for the user, you explained.

Reply:
so far, temperature is under `llm` key in the .toml, while the model_name is under the `prompt`[here](https://github.com/talkdai/dialog/blob/main/src/settings.py#L19). This PR makes both under `llm` key.